### PR TITLE
Followers feature

### PR DESCRIPTION
**Pull request by jshidalgokommit**
a. Follow/Unfollow other bloggers. 

b. Add comments only if I am following the blog author.

### CODE REVIEW

File: app/controllers/followers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class FollowersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @bloggers = User.all.where.not(id: current_user.id)
```
Comment (JoseCastroA) [inline app/controllers/followers_controller.rb:7]: @jshidalgokommit Why the variable is named `@bloggers`?

File: app/helpers/followers_helper.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module FollowersHelper
+
+  # render a follow or unfollow btn for specific user
+  def followers_btn(user)
+    if user_signed_in? && current_user.id != user.id
+      @@following = user.followed_by(current_user.id)
```
Comment (JoseCastroA) [inline app/helpers/followers_helper.rb:8]: @jshidalgokommit Why the double` @@`?

File: app/helpers/followers_helper.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  def follow_btn(user_to_follow)
+    form_with model: Follower, method: :post do |form|
+      res = form.hidden_field :followed_user_id, value: user_to_follow.id
+      res += form.hidden_field :followed_by_user_id, value: current_user.id
+      res += form.button '', class: 'btn badge bg-primary rounded-pill', title: 'Follow' do
+        'Follow <i data-feather="user-plus"></i>'.html_safe
+      end
+      res
+    end
+  end
```
Comment (JoseCastroA) [inline app/helpers/followers_helper.rb:28]: @jshidalgokommit Why are we putting frontend logic into a helper? If you need to do `DRY` (Don't repeat yourself), you can create a `_form` partial file. What do you think?

File: app/helpers/followers_helper.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  def unfollow_btn
+    form_with model: @@following, method: :delete do |form|
+      form.button '', class: 'btn badge bg-danger rounded-pill', title: 'Unfollow' do
+        'Unfollow <i data-feather="user-minus"></i>'.html_safe
+      end
+    end
+  end
```
Comment (JoseCastroA) [inline app/helpers/followers_helper.rb:36]: @jshidalgokommit Same here

**Review by JoseCastroA — CHANGES_REQUESTED** [review]

File: app/helpers/followers_helper.rb (hunk 2 above)
Comment (jshidalgokommit) [reply app/helpers/followers_helper.rb:8]: Sorry, I thought it was a variable only for the class. I should have used only @

File: app/helpers/followers_helper.rb (hunk 3 above)
Comment (jshidalgokommit) [reply app/helpers/followers_helper.rb:28]: Ok, helpers should not be used like this

**Comment by jshidalgokommit** [comment]
Hi @JoseCastroA, changes requests are applied

**Review by JoseCastroA — APPROVED** [review]

**Pull request merged**